Packs/McAfee-TIE/Integrations/McAfee-TIE/McAfee-TIE.py

```diff
@@ -280,308 +104,260 @@ def parse_reputation_human_readable(reputation):
     return res
 
 
-def parse_reference_human_readable(reference):
-    agent_guid = reference.get(FirstRefProp.SYSTEM_GUID)
-    date = reference.get(FirstRefProp.DATE)
+def parse_reference(reference):
+    agent_guid = reference.get('agentGuid')
+    date = reference.get('date')
+    try:
+        date = datetime.fromtimestamp(date)
+    except ValueError:
+        date = datetime.fromtimestamp(date / 1000)
 
     return {
-        'Date': epoch_to_localtime(date),
+        'Date': str(date),
         'AgentGuid': agent_guid.replace('{', '').replace('}', '')  # remove brackets if exist
     }
 
 
-def reputations_to_human_readable(reputations):
-    return [parse_reputation_human_readable(rep) for rep in reputations]
+def reputations_to_table(reputations):
+    return [parse_reputation(rep) for rep in reputations]
 
 
-def references_to_human_readable(references):
-    return [parse_reference_human_readable(ref) for ref in references]
+def references_to_table(references):
+    return [parse_reference(ref) for ref in references]
 
 
 def trust_level_to_score(trust_level):
-    if (trust_level >= TrustLevel.MIGHT_BE_TRUSTED):
-        return Common.DBotScore.GOOD
+    if (trust_level >= 70):
+        return 1
+    elif (trust_level == 30):
+        return 2
+    elif (trust_level == 0 or trust_level == 50):
+        return 0
+    elif (trust_level < 30):
+        return 3
+    else:
+        # Shouldn't reach here, as the API doesn't support 31-69 values except for 50)
+        return 0
 
-    elif (trust_level == TrustLevel.MIGHT_BE_MALICIOUS):
-        return Common.DBotScore.SUSPICIOUS
 
-    elif (trust_level == TrustLevel.NOT_SET or trust_level == TrustLevel.UNKNOWN):
-        return Common.DBotScore.NONE
+def get_thrust_level_and_score(reputations):
+    trust_level = 101  # more than the highst possible trust level
+    vendor = VENDOR_NAME
 
-    elif (trust_level < TrustLevel.MIGHT_BE_MALICIOUS):
-        return Common.DBotScore.BAD
+    for rep in reputations:
+        rep_trust_level = rep.get('trustLevel', 0)
+        if rep_trust_level != 0 and rep_trust_level < trust_level:
+            trust_level = rep.get('trustLevel')
+            vendor = get_provider(rep.get('providerId'))
```

Comment:
if this is only to find the minimum, you can do
min_rep_trust_level = min(rep.get('trustLevel') for rep in reputations if rep.get('trustLevel', 0) != 0 and rep.get('trustLevel', 0) < trust_level )